book_source/basic_users_guide/Getting-started.Rmd

```diff
@@ -87,6 +87,22 @@ Select the default drive volume type and click Next
     +	To TERMINATE the instance (which will DELETE your PEcAn machine), select your instance and click Actions > Instance state > Terminate. Terminated instances will not incur costs. In most cases you will also want to go to the Volumes menu and delete the storage associated with your PEcAn VM.Remember, AWS is free for one year, but will automatically charge a fee in second year if account is not cancelled.
 
 
+### Working with the PEcAn Containers (Docker) [Experimental]
+
+Following are the steps to setup a Docker instance of the PEcAn. This is experimental and can have a lot of bugs, If you find any bug please report it. 
+
+1. Make sure that the machine on which have docker and docker-compose installed. For instruction on how to install docker and docker-compose please visit the [official documentations](https://docs.docker.com/engine/installation/).
+
+2. Visit the PEcAn Project on [github](https://github.com/PecanProject/pecan/tree/develop) and clone the repository to your machine.
+
+3. cd to root of the repository and run `docker-compose up -d` here -d makes it run in detached mode so it won't show the log on the terminal
```

Comment:
As for when we use docker to release, i'd like to push it to 1.5.1. There are some pieces that need to be done (e.g. the ability to launch models in separate containers).

Releasing the models, there are a few options.
- We can release the docker file, which is like a makefile that people can then use to build their own containers. 
- We can ask for private repositories that we control who has access to it, default is $7/mo for 5 repos. Not sure if they have opensource options.
- We can setup our own registry, this requires some work and might make it harder for people to pull the images, we have full control and can have as many private as we want.